lib/tasks.ex

```diff
@@ -1,18 +1,6 @@
 defmodule Tasks do
-  @moduledoc """
-  Documentation for `Tasks`.
-  """
-
-  @doc """
-  Hello world.
-
-  ## Examples
-
-      iex> Tasks.hello()
-      :world
-
-  """
-  def hello do
-    :world
-  end
+  def fib(0), do: "There is no 0th fibonacci"
+  def fib(1), do: 0
+  def fib(2), do: 1
+  def fib(n), do: fib(n - 1) + fib(n - 2)
```

Comment:
While this is nice implementation of fibonacci, it does have one big problem :)

Try running this

```elixir
Tasks.fib(100000)
```